### ISSUE TITLE
[Bug Report] Safari 14.1: v-btn text color not changing when toggling disabled state

### ISSUE THREAD

**Issue by WIStudent** [vuetifyjs/vuetify#13599]
### Environment
**Vuetify Version:** 2.4.11
**Vue Version:** 2.6.12
**Browsers:** Safari 14.1 (16611.1.21.161.3)
**OS:** Mac OS 10.15.7

### Steps to reproduce
In Safari 14.1, switch between enabled/disabled, the text color will stay the same.

### Expected Behavior
Text color should change.
Screenshots from Chrome:
![Bildschirmfoto 2021-05-04 um 16 17 12](https://user-images.githubusercontent.com/2707930/117021464-a92b0300-acf7-11eb-9177-7d016954ffa8.png)
![Bildschirmfoto 2021-05-04 um 16 17 26](https://user-images.githubusercontent.com/2707930/117021469-ac25f380-acf7-11eb-9e38-641d009a00b0.png)

### Actual Behavior
Text color does not change.
Screenshots from Safari 14.1
![Bildschirmfoto 2021-05-04 um 16 18 37](https://user-images.githubusercontent.com/2707930/117021528-b7791f00-acf7-11eb-95f1-e2d47d05acfe.png)
![Bildschirmfoto 2021-05-04 um 16 18 47](https://user-images.githubusercontent.com/2707930/117021536-b9db7900-acf7-11eb-9dfc-08cc412b616a.png)

### Reproduction Link
<a href="https://codepen.io/wistudent/pen/xxqKdMQ" target="_blank">https://codepen.io/wistudent/pen/xxqKdMQ</a>

**Comment by KaelWD** [comment]
This needs to be reported to webkit

**Comment by WIStudent** [comment]
I think I found the corresponding issue in webkit: https://bugs.webkit.org/show_bug.cgi?id=225354

**Comment by WIStudent** [comment]
Someone in the webkit issue mentioned that it is a problem with `pointer-events`.

For anyone who does not want to wait for the next Safari update, I am using this as a workaround for now:
```
.v-btn--disabled {
  pointer-events: unset !important;
  cursor: default;
}
```

**Comment by KaelWD** [comment]
This seems to be fixed in 16611.1.21.161.6

**Comment by WIStudent** [comment]
I am closing this issue as it is fixed in the latest safari release on macOS and iOS 14.7
